samples/framework/libraries/migrate-library-csproj/src/Car/Car.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;net451;netstandard1.6</TargetFrameworks>
+    <AssemblyName>Car</AssemblyName>
+    <PackageId>Car</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <Reference Include="System.Collections.Concurrent" />
+    <Reference Include="System" />
+    <Reference Include="Microsoft.CSharp" />
+  </ItemGroup>
```

Comment:
For something like this, it might be more convenient to set it up with a negative conditional expression ...
```xml
<ItemGroup Condition=" '$(TargetFramework)' != 'netstandard1.6' ">
  <Reference Include="System.Collections.Concurrent" />
  <Reference Include="System" />
  <Reference Include="Microsoft.CSharp" />
</ItemGroup>
```
... if the `net45` and `net451` remain. Then, you only need this one `ItemGroup`.